.github/workflows/pr-release.yml

```
@@ -239,7 +239,7 @@ jobs:
           git config user.name "leanprover-community-mathlib4-bot"
           git config user.email "leanprover-community-mathlib4-bot@users.noreply.github.com"
 
-          if git branch -r | grep -q "nightly-testing-${MOST_RECENT_NIGHTLY}"; then
+          if git ls-remote --heads --tags origin | grep -q "refs/\(heads\|tags\)/nightly-testing-${MOST_RECENT_NIGHTLY}"; then
```

Not sure it matter, but with the ever-growing list of tags and heads, maybe
```suggestion
          if git ls-remote --heads --tags --exit-code origin "nightly-testing-${MOST_RECENT_NIGHTLY}" >/dev/null; then
```
instead?

Also file says “branch“ when it should be “tag“ in a few more places.